app/js/actions/QuestionActions.js

```diff
@@ -66,7 +66,17 @@ export function loadQuestions(page, tags) {
   let path = (tags && tags.length) ? '/questions/by_tags' : '/questions';
   return dispatch => {
     dispatch(displayLoader({shouldFetch: true}));
-    return webAPI(path, 'GET', {page, tags})
+    console.log(tags);
```

Comment:
@andela-iamadi please confirm that this approach works? Basically, the endpoint expects an array of tag_ids, `[1,2,3,4]`, I'm not sure if sending the array as a string works, because currently the strings are not being splitted on ZBE! If you look at the [documentation](https://github.com/zhishi-project/zhishi-backend/blob/master/docs/questions_endpoints.md#get-top_questions-1), you can see an approach that was taken to pass this! I'm guessing there are other approaches as well, but not confident that this works!